dev-utils/poly-cli/index.js

```diff
@@ -212,16 +151,21 @@ yargs(hideBin(process.argv))
         },
         handleCreate
     )
+    .version(false)
```

Comment:
Still. `version` is conventionally the CLI tool version. We probably shouldn't co-opt it for what we use it, which is the version for the feature we're generating. Anything that's not strictly version will do, like `fversion` or `initial-version`

Reply:
This is not a hack. I am just telling yargs that this command will not use `--version` as a standalone option. It does not make sense to ask for the version of the create command.

Reply:
You're right. This is not a general argument, it's a subcommand argument. It's also consistent with `inquirer`. But I have to say it's a hack, and a good one at that; one that I had not been able to find. So kudos :clap: